tests/topotests/lib/common_config.py

```diff
@@ -3236,36 +3231,34 @@ def verify_rib(
                     st_rt = str(ipaddress.ip_network(frr_unicode(st_rt)))
 
                     _addr_type = validate_ip_address(st_rt)
-                    if _addr_type != addr_type:
-                        continue
-
-                    if st_rt in rib_routes_json:
-                        st_found = True
-                        found_routes.append(st_rt)
+                    if _addr_type == addr_type:
```

Comment:
Same question as above: if this is just a re-indent, then ... why? and if not, if there's actually a meaningful change hidden in here, please highlight it and describe it.